models/org.go

```diff
@@ -50,7 +50,7 @@ func (org *User) GetOwnerTeam() (*Team, error) {
 }
 
 func (org *User) getTeams(e Engine) error {
-	return e.Where("org_id=?", org.ID).Find(&org.Teams)
+	return e.Where("org_id=?", org.ID).OrderBy("CASE WHEN name LIKE '" + OWNER_TEAM + "' THEN '' ELSE name END").Find(&org.Teams)
```

Comment:
Use proper SQL params instead of concatenating strings. Like this:

```go
OrderBy("name LIKE ?", variable)
```

Also, why `LIKE` and not `=`?